terraform-provider-constellation/internal/provider/attestation_data_source.go

```diff
@@ -70,13 +83,12 @@ func (d *AttestationDataSource) Schema(_ context.Context, _ datasource.SchemaReq
 	resp.Schema = schema.Schema{
 		// This description is used by the documentation generator and the language server.
 		MarkdownDescription: "The data source to fetch measurements from a configured cloud provider and image.",
```

Comment:
We might want to add a mention of the lock-step versioning behaviour if no explicit version is set to the description here.